osm2pgsql.lua

```diff
@@ -499,128 +499,120 @@ function osm2pgsql.process_node(object)
-        output_hstore['place:ES'] = 'region'
-        output_hstore['place:FR'] = 'region'
-        output_hstore['place:GB'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:IT'] = 'region'
-        output_hstore['place:JP'] = 'region'
-        output_hstore['place:KO'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NL'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
+        output_hstore['place'] = 'country'
         output_hstore['place:RU'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SE'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:US'] = 'region'
-        output_hstore['place:VN'] = 'region'
     end
 -- Turn off Somaliland label for most countries
     if object.tags.place and object.tags.wikidata == 'Q34754' then
-        output_hstore['place:AR'] = 'region'
-        output_hstore['place:BD'] = 'region'
-        output_hstore['place:BR'] = 'region'
-        output_hstore['place:CN'] = 'region'
-        output_hstore['place:EG'] = 'region'
-        output_hstore['place:GR'] = 'region'
-        output_hstore['place:ID'] = 'region'
-        output_hstore['place:IL'] = 'region'
-        output_hstore['place:IN'] = 'region'
-        output_hstore['place:MA'] = 'region'
-        output_hstore['place:NP'] = 'region'
-        output_hstore['place:PK'] = 'region'
-        output_hstore['place:PL'] = 'region'
-        output_hstore['place:PS'] = 'region'
-        output_hstore['place:PT'] = 'region'
-        output_hstore['place:RU'] = 'region'
-        output_hstore['place:SA'] = 'region'
-        output_hstore['place:SO'] = 'region'
-        output_hstore['place:TR'] = 'region'
-        output_hstore['place:TW'] = 'region'
-        output_hstore['place:UA'] = 'region'
-        output_hstore['place:VN'] = 'region'
+        output_hstore['place'] = 'country'
+    end
+
+-- Recast various dependencies as countries
+-- American Samoa
+    if object.tags.place == 'state' and object.tags['ISO3166-1'] == 'AS' then
+        output_hstore['place'] = 'country'
+    end
+-- Christmas Island
+    if object.tags.place == 'territory' and object.tags['ISO3166-1'] == 'CX' then
+        output_hstore['place'] = 'country'
+    end
+-- Cocos (Keeling) Islands
+    if object.tags.place == 'territory' and object.tags['ISO3166-1'] == 'CC' then
+        output_hstore['place'] = 'country'
+    end
+-- French Guiana
+    if object.tags.place == 'state' and object.tags['ISO3166-1'] == 'GF' then
+        output_hstore['place'] = 'country'
```

Comment:
This is an odd one, it's an overseas region of France (like Alaska and Hawaii), but also gets it's own ISO code.

My preference is to be true to the data and mark it **place** = `region`... and if you want to add POV overrides for ISO and all the other countries we're tracking to be `country` then that's OK.

The full set are ([src](https://en.wikipedia.org/wiki/Overseas_France#Overseas_regions_and_departments)):

- French Guiana
- Guadeloupe
- Martinique
- Mayotte
- Réunion

Reply:
Changed them to region

Reply:
(We keep bouncing back and forth on this one... I left new comments farther down.)